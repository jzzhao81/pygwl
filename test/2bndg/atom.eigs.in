       9       1     1.0000000000     0.0000000000    1000
       5       2     1.0000000000     0.0000000000    0100
       1       3     1.0000000000     0.0000000000    0000
       3       4     1.0000000000     0.0000000000    0010
       2       5     1.0000000000     0.0000000000    0001
      13       6     1.0000000000     0.0000000000    1100
      11       7     1.0000000000     0.0000000000    1010
       7       8     1.0000000000     0.0000000000    0110
      10       9     1.0000000000     0.0000000000    1001
       6      10     1.0000000000     0.0000000000    0101
       4      11     1.0000000000     0.0000000000    0011
      15      12     1.0000000000     0.0000000000    1110
      14      13     1.0000000000     0.0000000000    1101
      12      14     1.0000000000     0.0000000000    1011
       8      15     1.0000000000     0.0000000000    0111
      16      16     1.0000000000     0.0000000000    1111
