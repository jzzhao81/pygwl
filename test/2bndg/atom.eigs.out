      1       -0.1000000000
      2       -0.1000000000
      3        0.0000000000
      4        0.1000000000
      5        0.1000000000
      6        0.7000000000
      7        0.7000000000
      8        0.7000000000
      9        0.7763932023
     10        0.9000000000
     11        1.2236067978
     12        2.4000000000
     13        2.4000000000
     14        2.6000000000
     15        2.6000000000
     16        5.0000000000
